themes/default/assets/config/tailwind.config.js

```diff
@@ -165,22 +177,16 @@ module.exports = {
             violet,
         },
         maxHeight: {
-            '25': '25vh',
-            '50': '50vh',
-            '75': '75vh',
-            '100': '100vh',
-        }
+            "25": "25vh",
+            "50": "50vh",
+            "75": "75vh",
+            "100": "100vh",
+        },
+        screens: {
```

Comment:
v2 also adds a `2xl` container width, which messes with our overall layout. Declaring these explicitly (https://tailwindcss.com/docs/upgrading-to-v2#configure-your-breakpoints-explicitly) gives us the v1 settings, and then in the Docs, where we do want a wider layout, we apply that using the v2 breakpoint. (Also in this PR.)